code/processes/dqe.q

```diff
@@ -1,17 +1,19 @@
 \d .dqe
 
 configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqeconfig.csv"]];
+detailcsv:@[value;`.dqe.detailcsv;first .proc.getconfigfile["dqedetailtab.csv"]];
 
 init:{
   .lg.o[`init;"searching for servers"];
   .servers.startup[];                                                                                           /- Open connection to discovery
+  {.[.api.add;value x]}'[.dqe.readdqeconfig[.dqe.detailcsv;"SB***"]];                                           /- add dqe functions to .api.detail
   }
 
 configtable:([] action:`$(); params:(); proctype:`$(); procname:`$(); mode:`$(); starttime:`timestamp$(); endtime:`timestamp$(); period:`timespan$())
 
-readdqeconfig:{[file]
+readdqeconfig:{[file;types]
```

Comment:
Is there a generic TorQ csv parser?

Reply:
I was not able to find a generic TorQ csv parser.